### ISSUE TITLE
Calculate accuracy for NER multilabel classification

### ISSUE THREAD

**Issue by roboserg** [ThilinaRajapakse/simpletransformers#883]
model.eval_model() does not give me a simple accuracy metric. Is there any way to get it? I tried doing it manually but for some reason, the number of samples was different from true and pred.

```
true = test_df['labels'].tolist()

lst = []
for arr in model_outputs:
    for y in arr:
        lst.append(np.argmax(arr))

predicted = lst

from sklearn.metrics import accuracy_score
accuracy_score(true, predicted)

**Found input variables with inconsistent numbers of samples: [31422, 31417]**
```

**Comment by ThilinaRajapakse** [comment]
[Docs](https://simpletransformers.ai/docs/tips-and-tricks/#additional-evaluation-metrics)

**Comment by roboserg** [comment]
Thanks
